script/discourse

```diff
@@ -91,12 +102,16 @@ class DiscourseCLI < Thor
     end
 
     old_include_s3_uploads_in_backups = SiteSetting.include_s3_uploads_in_backups
-    SiteSetting.include_s3_uploads_in_backups = true if options[:s3_uploads]
+    SiteSetting.include_s3_uploads_in_backups = true if !options[:sql_only] && options[:s3_uploads]
```

Comment:
```suggestion
    SiteSetting.include_s3_uploads_in_backups = !options[:sql_only] && options[:s3_uploads]
```

Reply:
Currently, if `include_s3_uploads_in_backups` is somehow already enabled, s3 uploads will be included without explicitly passing the `--s3-uploads` option. 

With this short-circuit, it will disable `include_s3_uploads_in_backups` if it's already enabled and the `--s3-uploads` option isn't passed.  Do we want that here?

Reply:
It's already being reset in:

https://github.com/discourse/discourse/blob/f67af10342321ecf82e0024243b748ea34bf22c7/script/discourse#L138-L140

Reply:
Ah, right! Sorry, I looked only at the diff :blush: